### ISSUE TITLE
oc-client renderComponent(s) fail with 'getCompiledTemplate is not a function'

### ISSUE THREAD

**Issue by nenodimitrov** [opentable/oc#387]
I am using oc-client 2.1.0 in a nodejs project and when I call `renderComponent` I get the following error:

```
/node_modules/oc-client/src/get-compiled-template.js:36
        cb(null, templateEngines[template.type].getCompiledTemplate(templateText, template.key));
                                                ^

TypeError: templateEngines[template.type].getCompiledTemplate is not a function
    at /Users/nenodimitrov/projects/partner-value/branded-seo-generator/server/node_modules/oc-client/src/get-compiled-template.js:36:49
    at respond (/Users/nenodimitrov/projects/partner-value/branded-seo-generator/server/node_modules/minimal-request/src/index.js:35:7)
    at IncomingMessage.<anonymous> (/Users/nenodimitrov/projects/partner-value/branded-seo-generator/server/node_modules/minimal-request/src/index.js:73:11)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

It looks like while the source for oc-client is written for os-template-jade 2.0, the dependency in package.json is still oc-template-jade 1.0.0 thus the `templateEngine` has `getPrecompiledTemplate` method instead of `getCompiledTemplate`

**Comment by matteofigus** [comment]
You're right. @nickbalestra I think we didn't update the client/package.json when working on this https://github.com/opentable/oc/pull/385

And the tests didn't catch the issue as they use the root's package.json as source.

**Comment by matteofigus** [comment]
@nenodimitrov I just published 2.1.1 with the fix. Sorry for the inconvenience.
